mp/game/neo/cfg/user_default.scr

```diff
@@ -0,0 +1,116 @@
+// NOTE:  THIS FILE IS AUTOMATICALLY REGENERATED, 
+//DO NOT EDIT THIS HEADER, YOUR COMMENTS WILL BE LOST IF YOU DO
+// User options script
+//
+// Format:
+//  Version [float]
+//  Options description followed by 
+//  Options defaults
+//
+// Option description syntax:
+//
+//  "cvar" { "Prompt" { type [ type info ] } { default } }
+//
+//  type = 
+//   BOOL   (a yes/no toggle)
+//   STRING
+//   NUMBER
+//   LIST
+//
+// type info:
+// BOOL                 no type info
+// NUMBER       min max range, use -1 -1 for no limits
+// STRING       no type info
+// LIST          delimited list of options value pairs
+//
+//
+// default depends on type
+// BOOL is "0" or "1"
+// NUMBER is "value"
+// STRING is "value"
+// LIST is "index", where index "0" is the first element of the list
+
+
+// Half-Life User Info Configuration Layout Script (stores last settings chosen, too)
+// File generated:  Tue Jul 13 17:43:58 AM
+//
+//
+// Cvar	-	Setting
+
+VERSION 1.0
+
+DESCRIPTION INFO_OPTIONS
+{
+	"neo_name"
+	{
+		"Fake Name"
+		{ STRING }
+		{ "Neotokyo Noob" }
+	}
+	"cl_onlysteamnick"
+	{
+		"Only use Steam names"
+		{ BOOL }
+		{ 1 }
+	}
```

Comment:
Default `neo_name` change to "". Player's Steam name will be used when empty even if `cl_onlysteamnick=0`

Likewise as the other suggestion, flip `cl_onlysteamnick` default here also.